src/main/java/edu/hm/hafner/analysis/parser/AquaScannerParser.java

```diff
@@ -0,0 +1,92 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.text.MessageFormat;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Parser for Aqua Scanner CLI (scannercli) tool.
+ *
+ * @author Juri Duval
+ */
+public class AquaScannerParser extends JsonIssueParser {
+    private static final String VALUE_NOT_SET = "-";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_SENSITIVE = "sensitive";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_MALWARE = "malware";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_CRITICAL = "critical";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_HIGH = "high";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_MEDIUM = "medium";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_LOW = "low";
+    private static final String AQUA_VULNERABILITY_LEVEL_TAG_NEGLIGIBLE = "negligible";
+    private static final long serialVersionUID = 1L;
+
+    @Override
+    protected void parseJsonObject(final Report report, final JSONObject jsonReport, final IssueBuilder issueBuilder) {
+        JSONArray resources = jsonReport.optJSONArray("resources");
+        if (resources != null) {
+            parseResources(report, resources, issueBuilder);
+        }
+    }
+
+    private void parseResources(final Report report, final JSONArray resources, final IssueBuilder issueBuilder) {
+        for (int i = 0; i < resources.length(); i++) {
+            JSONObject resourceWrapper = (JSONObject) resources.get(i);
+            if (!resourceWrapper.isNull("vulnerabilities") && !resourceWrapper.isNull("resource")) {
+                JSONObject resource = resourceWrapper.getJSONObject("resource");
+                JSONArray vulnerabilities = resourceWrapper.getJSONArray("vulnerabilities");
+                for (Object vulnerability : vulnerabilities) {
+                    report.add(convertToIssue(resource, (JSONObject) vulnerability, issueBuilder));
+                }
+            }
+        }
+    }
+
+    private Issue convertToIssue(final JSONObject resource, final JSONObject vulnerability,
+            final IssueBuilder issueBuilder) {
+        final String fileName = resource.isNull("path") ? resource.optString("name", VALUE_NOT_SET)
+                : resource.getString("path");
+        return issueBuilder
+                .setFileName(fileName)
+                .setSeverity(mapSeverity(vulnerability.optString("aqua_severity", "UNKNOWN")))
+                .setMessage(vulnerability.optString("name", "UNKNOWN"))
+                .setDescription(formatDescription(fileName, resource, vulnerability))
+                .buildAndClean();
+    }
+
+    @SuppressFBWarnings("IMPROPER_UNICODE")
```

Comment:
Not required if used in conjunction with `StringContainsUtils.containsAnyIgnoreCase`.

Reply:
Removed.